flexmeasures/utils/app_utils.py

```diff
@@ -63,3 +65,35 @@ def set_secret_key(app, filename="secret_key"):
         )
 
         sys.exit(2)
+
+
+def register_plugin(app: Flask):
+    """
+    Register a FlexMeasures plugin as Blueprint.
+
+    Assumptions:
+    - We'll use the name of your plugin folder as the name.
+    - Your plugin folder contains an 'fmplugin' folder with an __init__.py file.
+    - In this init, you define a Blueprint object called <plugin folder>_bp
+
+    TODO: Support multiple plugins.
```

Comment:
Probably. But I also would need to test that which is somewhat more effort. Actually, I believe it might take a while until someone needs it, or do you disagree?

Reply:
Hopefully not a lot of extra effort: I suggest the test would be to split up your one module defining both a UI view and CLI function into two modules. We'd indeed most likely be the first users of this plugin architecture, but I would prefer us to work in a modular fashion on small and dedicated plugins from the start, if possible. But fair enough, if your preference is to have multiple plugins be outside of the scope of this PR, let's open a separate ticket.

Reply:
In my eyes, one plugin and one private repo is what one would usually make per customer, and that's what we'll be doing for a few months. Of course, if our customers want custom stuff per client of theirs, then we should have more than one plugin per server. But that would also require our account data model (multiple accounts per server).